dataminer/Administrator_guide/DataMiner_Systems/BPA_tests/BPA_LargeAlarmTrees.md

```diff
@@ -0,0 +1,57 @@
+---
+uid: BPA_LargeAlarmTrees
+---
+
+# Large Alarm Trees
+
+This BPA test is executed on one DMA per cluster. It will fetch active alarm trees and check if any are getting too large.
+If that's the case you'll have to take corrective actions, as suggested [here](https://aka.dataminer.services/LargeAlarmTrees).
```

Comment:
For links within docs, it's better to use links in xref format, as these require the least maintenance. I'll adjust this here. However, note also that the aka link you added here does not work; it should be `http://aka.dataminer.services/largealarmtrees`.